R/susie_new.R

```diff
@@ -0,0 +1,147 @@
+susie_new = function (X, y, L = min(10,ncol(X)),
```

Comment:
you can delete the obsolete R files and only keep the updated ones, so you dn't have name conflicts. On github you can always check them in history